Monika After Story/game/definitions.rpy

```diff
@@ -6099,23 +6099,17 @@ init -1 python:
         for savegame in renpy.list_saved_games(fast=True):
             renpy.unlink_save(savegame)
 
-
-    def delete_character(name):
+    def mas_delete_all_chrs(log=False):
         """
-        Deletes a .chr file for a character
+        Deletes all chr files under /characters/ folder.
 
         IN:
-            name of the character who's chr file we want to delete
+            log (deprecated):
+                If True, print log message on deletion error. Deprecated and is
+                no longer used as docking station will always log errors.
```

Comment:
Since this func is changing entirely, we can probably remove the `log` param and the deprecation of the param